deviations/views.py

```diff
@@ -26,14 +72,18 @@ class AddDeadlinesView(CourseInstanceMixin, BaseFormView):
     access_mode = ACCESS.TEACHER
     template_name = "deviations/add_dl.html"
     form_class = DeadlineRuleDeviationForm
+    already_have_deviation = []
```

Comment:
What is the purpose of this class variable `already_have_deviation`? Why is it a list and later you do `            self.already_have_deviation = True`?